test/catalogd-upgrade-e2e/unpack_test.go

```diff
@@ -21,7 +21,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	catalogdv1 "github.com/operator-framework/operator-controller/catalogd/api/v1"
-	catalogde2e "github.com/operator-framework/operator-controller/test/catalogd-e2e"
+	"github.com/operator-framework/operator-controller/test/utils"
```

Comment:
Same as above

nitpick : can we import this as testutil? because we already have internal/util in code.